### PR TITLE
[MM-47963] Import/Export for topical threads

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
Add import/export functionality for topical threads

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-47963

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by isacikgoz** [comment]
I want to notify our import/export expert here to cast an eye :) cc @noxer

File: app/import_functions.go [hunk 1]
```diff
@@ -1855,6 +1830,110 @@ func (a *App) importMultipleDirectPostLines(c request.CTX, lines []imports.LineI
 	return 0, nil
 }
 
+func (a *App) getPostWithAttachments(c request.CTX, postData *imports.PostImportData, userId, teamId, channelId string) (*model.Post, *model.AppError) {
+	// Check if this post already exists.
+	posts, nErr := a.Srv().Store().Post().GetPostsCreatedAt("", *postData.CreateAt)
```
Comment (isacikgoz) [inline app/import_functions.go:1835]: Is it intentional that we are not querying with channel id?

**Review by isacikgoz — COMMENTED** [review]
Looks good overall, haven't look at the queries yet but at glance I have a question and a nit.

**Comment by lieut-data** [comment]
We're hitting pause on topical threads as originally envisioned, at least for now.

**Pull request closed without merging**